### PR TITLE
Created Round View and added minor modifications to other views

### PR DESCRIPTION
**Pull request by ethanmcf**
PR updates:
- Created for round: view, view model, state
- Implemented main file to run views
- Added functionality of game settings view so clicking play game button takes you to round view
- Created GameOverController and injected controller, view model and view manager model in game over view

Below is the round view. Updates will need to be made to this since we will have to differentiate when we are using multiple choice vs text input. Possibly having a visual update for the play song button for play/stop will need to be implemented later on when play and stop audio functionality is implemented.

Screenshot 2023-11-05 at 2 13 26 PM

### CODE REVIEW

**Review by rayceramsay — COMMENTED** [review]
We'll definitely have to refactor some of the things you mentioned later but this serves our purposes well for right now!

Left a couple of notes for later and one discussion question that I wouldn't mind chatting about (i.e. the controller comment). Once we touch on that this PR has my approval 👍

File: src/view/RoundView.java
Comment (ethanmcf) [reply src/view/RoundView.java:35]: Yup totally agree great to note so we can remember for later

File: src/interface_adapter/round/RoundController.java
Comment (ethanmcf) [reply src/interface_adapter/round/RoundController.java:7]: I also agree, since technically right now the views aren't dependant on the controllers because there is no interaction occurring. Not sure what you guys think as well but I believe Rayce is right and we may want to delete these controllers for now until we start doing our use cases.

**Review by rayceramsay — APPROVED** [review]
Looks great!

**Review by NickinAction — APPROVED** [review]

**Pull request merged**
